src/main/components/uml-element/assessable/assessable.tsx

```diff
@@ -151,10 +151,13 @@ export const assessable = (
           console.error('Could not parse artemis sgi', e);
           return;
         }
+        // TODO: following messages should be received from Artemis
```

Comment:
Is this part of future work or is to be removed and you forgot about it?

Reply:
yes, I left this comment as a reminder.